docs/source/strategy.rst

```diff
@@ -64,3 +64,9 @@ The following processing parameters my be set:
     do_check
         Boolean flag that determines whether the check for self-calibration convergence should be done for this cycle.
 
+    convergence_ratio
+        Float that sets the minimum ratio of the current image noise to the previous image noise above which selfcal is considered to have converged (must be in the range 0.5 -- 2).
```

Comment:
Out of curiosity. But if convergence_ratio is larger than 1, you're no longer converging, right?

Reply:
Yes, that's quite true, but sometimes (at least in Factor) one iteration can have a little higher noise than the preceding one even though the whole of selfcal is still converging (e.g. the noise is worse but dynamic range increases -- it would probably to good to add further properties to check but for a start the noise is the most important I think). So one might want to let it continue even if the ratio is above 1.